src/components/switches/checkbox/checkbox.spec.js

```diff
@@ -84,6 +84,36 @@ describe('<Checkbox>', () => {
         });
       });
 
+      describe('when hovered', () => {
+        beforeEach(() => {
+          props = createTestProps({
+            isHovered: true,
+          });
+
+          wrapper = shallow(<Checkbox {...props} />);
+        });
+
+        it('should render checkbox in a hovered state', () => {
+          expect(wrapper.exists('.isHovered')).toBe(true);
```

Comment:
I would use `toHaveClassName` instead as It gives nicer error messages.. or maybe I'm missing some upsite of doing it like this?

Reply:
I find `exists('.isHovered')` to give less coupling to component internals. It just check that some traces of hovered state are there, without bounding itself to specific DOM-element which is accepting this class.

Reply:
Ah that's what I thought! A drop on a hot stone (a badly translated german saying :D) in this case unfortunately, but a good thought! Can't wait to convert these to `react-testing-library` and VRTs.

Another hint would be to import the styles and then do `styles.isHovered`. This helps with the logic when reading the tests. But I don't care as this is going away. Just wanted to point it out in case you weren't aware already.